guides/v2.2/extension-dev-guide/plugins.md

```diff
@@ -102,7 +102,7 @@ After methods have access to all the arguments of their observed methods. When t
 
 Below is an example of an after method that accepts the `null` result and arguments from the observed `login` method for [`Magento\Backend\Model\Auth`]({{ site.mage2100url }}app/code/Magento/Backend/Model/Auth.php){:target="_blank"}:
 
-``` PHP
+```php
```

Comment:
are these in italics? Maybe my eyes are deceiving me. Not sure if this affects any formatting.

Reply:
Seems like it is but it would need markdown to really be...